package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.4.1",
+    "dotenv": "^8.2.0",
```

Comment:
this is incorrect. please move this to `devDependencies`. You should have run this -`-save-dev`

Reply:
solve it!